lib/clever-ruby/api_operations/page.rb

```diff
@@ -66,6 +67,29 @@ def last(num = nil)
         return @all.last num if num
         @all.last
       end
+
+      private
+
+      # District_admins API returns slightly different response, so need to standardize it
+      # @api private
+      # @returns response with each elem in [:data] containing a hash with :data and :uri keys,
+      #   unless already in that format
+      # @example
+      #   input:
+      #     {data: [{id: 1, email: 'admin1@school.edu'},
+      #             {id: 2, email: 'admin2@school.edu'}]}
+      #   output:
+      #     {data: [{data: {id: 1, email: 'admin1@school.edu'}, uri: '/v1.1/district_admins/1'},
+      #             {data: {id: 2, email: 'admin2@school.edu'}, uri: '/v1.1/district_admins/2'}]}
+      def standardize_response(response)
```

Comment:
ugh... should've read the documentation closer... I'll probably just do a new PR with that approach instead.